crates/bevy_app/src/ci_testing.rs

```diff
@@ -2,10 +2,10 @@ use serde::Deserialize;
 
 use crate::{app::AppExit, App};
 
-/// Configuration for automated testing on CI
+/// A configuration for automated testing on CI.
```

Comment:
```suggestion
/// A configuration struct for automated CI testing.
```

Reply:
This could really use some real docs if we're keeping it pub.

Reply:
I think this should be `pub(crate)`. It is used in the `setup_app` function and gets its data from the `CI_TESTING_CONFIG` env var or the `ci_testing_config.ron`. So you pretty much can't really change it other than accessing the resource in another system (like `ci_testing_exit_after` does). Probably the only reason to leave it `pub` would be to be able to use it in systems and do something with it (whatever that'd be).

Either way I'm not going to convert this PR into a breaking change. If we want it to be `pub(crate)` we could make a separate PR and discuss if it should be `pub` or `pub(crate)`.

Reply:
Agreed; just leave it for now.